web/src/beta/components/fields/ToggleField/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
Don't need React imports. Vite uses a plugin that injects them on build/runtime

Reply:
Vite sounds nice.